build.gradle.kts

```diff
@@ -76,6 +76,15 @@ java {
     withSourcesJar()
 }
 
+tasks.withType<Jar>().configureEach {
+    manifest {
+        attributes(
+            "Implementation-Title" to "Gradle Profiler",
+            "Implementation-Version" to archiveVersion
```

Comment:
I think it would be better to use `project.version` here, since we may remove the version from the archive name.

Reply:
OK, fixed.